torch/optim/lr_scheduler.py

```diff
@@ -700,6 +700,55 @@ def load_state_dict(self, state_dict):
             self._schedulers[idx].load_state_dict(s)
 
 
+class PolynomialLR(_LRScheduler):
+    """Decays the learning rate of each parameter group using a polynomial function
+    in the given total_iters. When last_epoch=-1, sets initial lr as lr.
+
+    Args:
+        optimizer (Optimizer): Wrapped optimizer.
+        total_iters (int): The number of steps that the scheduler decays the learning rate. Default: 5.
+        power (int): The power of the polynomial. Default: 1.0.
+        verbose (bool): If ``True``, prints a message to stdout for
+            each update. Default: ``False``.
+
+    Example:
+        >>> # Assuming optimizer uses lr = 0.001 for all groups
+        >>> # lr = 0.00010   if epoch == 0
```

Comment:
Should this be `0.001` ?

Reply:
Ah that's a remnant from https://github.com/pytorch/pytorch/pull/82769#discussion_r937552195

We should change it to `0.00010`. cc @federicopozzi33

Reply:
> Ah that's a remnant from [#82769 (comment)](https://github.com/pytorch/pytorch/pull/82769#discussion_r937552195)
> 
> We should change the lr on the sentence to `0.0001`. cc @federicopozzi33

The `lr` on the sentence (_Assuming optimizer uses lr = 0.001 for all groups_) is right. The `lr` at the first epoch was wrong (before `0.0001`, now `0.001`).